src/platforms/react-native/performance/included-instrumentation.mdx

```diff
@@ -73,13 +128,33 @@ const App = () => {
     </NavigationContainer>
   );
 };
+
+// Class Component Example
+class App extends React.Component {
+  // Create a ref for the navigation container
+  navigation = React.createRef();
+
+  componentDidMount() {
+    // Register the navigation container with the instrumentation
+    routingInstrumentation.registerNavigationContainer(navigation);
+  }
+
+  render() {
+    return (
+      // Connect the ref to the navigation container
+      <NavigationContainer ref={this.navigation}>
+        ...
+      </NavigationContainer>
+    )
+  }
+}
 ```
 
 ### React Navigation V4
 
 Note that this routing instrumentation will create a transaction on every route change including `goBack` navigations.
 
-```js
+```javascript
```

Comment:
does this improve something? I see both patterns everywhere

Reply:
Yes, on the code block previews it'll say Javascript instead of Js
<img width="192" alt="Screen Shot 2021-02-03 at 3 42 39 PM" src="https://user-images.githubusercontent.com/30991498/106720784-7446db80-6636-11eb-94d2-ed6df10e9fa1.png">